ui/components/ui/nft-info/nft-info.js

```diff
@@ -40,16 +38,15 @@ export default function NftInfo({ assetName, tokenAddress, tokenId }) {
         </Box>
       </Box>
       <Box marginTop={4} marginRight={4}>
-        <Button className="nft-info__button" type="link">
+        <ButtonLink className="nft-info__button">
```

Comment:
using new `Button` component from component-library does not work hence the `ButtonLink`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/6ac63200-789d-455d-a97a-b06b9b90d5a4)